docker/playwright-E2E-tests-multi-node.yml

```
@@ -0,0 +1,140 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Playwright setup for multi-node
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+  artemis-app-node-1:
+    &artemis-app-base
+    image: ghcr.io/ls1intum/artemis:${ARTEMIS_DOCKER_TAG:-latest}
+    extends:
+      file: ./artemis.yml
+      service: artemis-app
+    pull_policy: always
+    restart: always
+    user: 0:0
+    depends_on:
+      mysql:
+        condition: service_healthy
+      jhipster-registry:
+        condition: service_healthy
+      activemq-broker:
+        condition: service_healthy
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    container_name: artemis-app-node-1
+    env_file:
+      - ./artemis/config/prod-multinode.env
+      - ./artemis/config/node1.env
+      - ./artemis/config/cypress.env
+
+  artemis-app-node-2:
+    <<: *artemis-app-base
+    container_name: artemis-app-node-2
+    depends_on:
+      mysql:
+        condition: service_healthy
+      jhipster-registry:
+        condition: service_healthy
+      activemq-broker:
+        condition: service_healthy
+      artemis-app-node-1:
+        condition: service_healthy
+    env_file:
+      - ./artemis/config/prod-multinode.env
+      - ./artemis/config/node2.env
+      - ./artemis/config/cypress.env
+
+  artemis-app-node-3:
+    <<: *artemis-app-base
+    container_name: artemis-app-node-3
+    depends_on:
+      mysql:
+        condition: service_healthy
+      jhipster-registry:
+        condition: service_healthy
+      activemq-broker:
+        condition: service_healthy
+      artemis-app-node-1:
+        condition: service_healthy
```

Maybe another anchor could be useful?

```suggestion
    depends_on:
      &depends-on-base
      mysql:
        condition: service_healthy
      jhipster-registry:
        condition: service_healthy
      activemq-broker:
        condition: service_healthy
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    container_name: artemis-app-node-1
    env_file:
      - ./artemis/config/prod-multinode.env
      - ./artemis/config/node1.env
      - ./artemis/config/cypress.env

  artemis-app-node-2:
    <<: *artemis-app-base
    container_name: artemis-app-node-2
    depends_on:
      <<: *depends-on-base
      artemis-app-node-1:
        condition: service_healthy
    env_file:
      - ./artemis/config/prod-multinode.env
      - ./artemis/config/node2.env
      - ./artemis/config/cypress.env

  artemis-app-node-3:
    <<: *artemis-app-base
    container_name: artemis-app-node-3
    depends_on:
      <<: *depends-on-base
      artemis-app-node-1:
        condition: service_healthy
```